docs/pages/build/eas-json.mdx

```diff
@@ -158,6 +158,31 @@ It's common to want to share build tool configuration between profiles, and we c
 }
 ```
 
+### Selecting resource class
+
+If your project requires extensive resources due to its complexity or size or if you want your builds to run faster, you can choose to run your jobs on `large` workers, which have access to a more powerful CPU and bigger memory. For more details on class-specific resources, see [Android](/eas/json/#resourceclass-1) and [iOS](/eas/json/#resourceclass-2) `resourceClass` property documentations.
```

Comment:
```suggestion
If your project requires extensive resources due to its complexity or size, or if you want your builds to run faster, you can use `large` workers to run your jobs. These workers have access to a more powerful CPU and bigger memory.
```

- Split the sentence as it was too long
- Remove the link to the resourceClass documentation and instead add those links in the next line when referencing each platform for what they are.
- Also, there is something missing from this section. In the example below, we providing examples for both `medium` and `large` but the section reads like it is only meant to focus on `large`. I'd suggest adding some context for normal resource class (as to what it is and what is the default class -- `medium`). This will make things clear instead of raising a confusion.

Reply:
Hmm, I think with links moved to `android.resourceClass` they lose the hint of providing more details (what exactly CPU / memory do people get when selecting which resource class), which is what I was going for here. Let me think what to do here

Reply:
How about this

https://github.com/expo/expo/blob/a0341402af5ee837784a5a6a94b46aad2dc3eabc/docs/pages/build/eas-json.mdx#L165